src/Resources/config/services.xml

```diff
@@ -73,5 +76,9 @@
         <service id="Webgriffe\SyliusClerkPlugin\Service\PublicApiKeyProvider">
             <argument/>
         </service>
+
+        <service id="webgriffe_sylius_clerk.resolver.channel_enabled" class="Webgriffe\SyliusClerkPlugin\Service\ChannelApiKeyChecker">
```

Comment:
The class is called "checker" but the service is called "resolver". Given the fact that the interface returns a boolean I prefer the name "checker"

Reply:
Whops!